src/components/NewMessageDetail.vue

```diff
@@ -82,11 +82,22 @@ export default {
 					subject = buildForwardSubject(message.subject)
 					msgTo = []
 				}
+				
+				// Remove original recipient from 'To' and 'Cc' fields
+				// This only works if original recipient's email address is the same as the email
+				// address of the account for which the original email was received, not an alias.
+				var account = this.$store.getters.getAccount(message.accountId)
```

Comment:
For replies we have the `buildRecipients` helper in `ReplyBuilder.js`. What about doing the same for forward? If we extract this little logic into an external function we can unit test it :)

Reply:
Nice :-)

I'll have a look :-)

Reply:
https://github.com/nextcloud/mail/blob/master/src/ReplyBuilder.js and https://github.com/nextcloud/mail/blob/master/src/tests/unit/ReplyBuilder.spec.js are the files. You can run the js tests with `npm run test:watch` during development and see the results almost instantly!

Reply:
this utiliy doesn't seem so be used:

```
====> function "buildRecipients" not used

root@vps612142:~/mail# grep -R buildRecipients src/
src/ReplyBuilder.js:export const buildRecipients = (envelope, ownAddress) => {
src/tests/unit/ReplyBuilder.spec.js:import {buildReplyBody, buildRecipients, buildReplySubject} from '../../ReplyBuilder'
src/tests/unit/ReplyBuilder.spec.js:            const reply = buildRecipients(envelope, a)
src/tests/unit/ReplyBuilder.spec.js:            var reply = buildRecipients(envelope, b)
src/tests/unit/ReplyBuilder.spec.js:            const reply = buildRecipients(envelope, b)
src/tests/unit/ReplyBuilder.spec.js:            const reply = buildRecipients(envelope, d)
src/tests/unit/ReplyBuilder.spec.js:            const reply = buildRecipients(envelope, e)
src/tests/unit/ReplyBuilder.spec.js:            const reply = buildRecipients(envelope, me)
src/tests/unit/ReplyBuilder.spec.js:            const reply = buildRecipients(envelope, jan)
src/components/Message.vue:import {buildRecipients as buildReplyRecipients, buildReplySubject} from '../ReplyBuilder'


====> function "buildReplyRecipients" used but not defined !?!

root@vps612142:~/mail# grep -R buildReplyRecipients src/
src/components/Message.vue:import {buildRecipients as buildReplyRecipients, buildReplySubject} from '../ReplyBuilder'
src/components/Message.vue:                                     this.replyRecipient = buildReplyRecipients(message, {
root@vps612142:~/mail# vi src/ReplyBuilder.js
root@vps612142:~/mail#
```

Reply:
"import {buildRecipients as buildReplyRecipients" aliased ;)

Reply:
oups by bad :-)